File: CyberSecurity/Java/ComputerSystem.java
public class ComputerSystem {
    private String systemName;
    private String ipAddress;
    private String systemType;
    protected String location;
    public String status;

    public ComputerSystem() {
        this.systemName = "Unknown";
        this.ipAddress = "0.0.0.0";
        this.systemType = "Generic";
        this.location = "Unknown";
        this.status = "Active";
    }

    public ComputerSystem(String name) {
        this.systemName = name;
        this.ipAddress = "192.168.1.1";
        this.systemType = "Generic";
        this.location = "Office";
        this.status = "Active";
    }

    public ComputerSystem(String name, String ip, String type) {
        this.systemName = name;
        this.ipAddress = ip;
        this.systemType = type;
        this.location = "Office";
        this.status = "Active";
    }

    public String getSystemName() {
        return systemName;
    }

    public String getIpAddress() {
        return ipAddress;
    }

    public void setIpAddress(String ip) {
        this.ipAddress = ip;
    }

    public String getSystemType() {
        return systemType;
    }

    public void displayInfo() {
        System.out.println("System: " + systemName + " | IP: " + ipAddress + " | Type: " + systemType);
    }
}

File: CyberSecurity/Python/computer_system.py
class ComputerSystem:
    def __init__(self, system_name, ip_address=None):
        self.system_name = system_name
        self.ip_address = ip_address
        self.is_active = True

    def display_system_name(self):
        print(f"System Name: {self.system_name}")

    def register_ip_address(self, ip_address):
        self.ip_address = ip_address
        print(f"IP Address registered: {ip_address}")

    def delete_system(self):
        self.is_active = False
        print(f"{self.system_name} deleted.")
